src/core.c

```diff
@@ -2614,13 +1538,5 @@ int emulate(x64emu_t* emu, elfheader_t* elf_header)
         trace_func = NULL;
     }
 #endif
-
-#ifdef DYNAREC
-    if (box64_dynarec_perf_map && box64_dynarec_perf_map_fd != -1) {
```

Comment:
but the close part of this, were it is now?

Reply:
oops, this is a mistake.